pkg/lightning/backend/external/engine.go

```diff
@@ -717,7 +717,7 @@ func (m *MemoryIngestData) NewIter(
 	}
 }
 
-// GetTS implements IngestData.GetTS.
+// GetTS implements IngestData.GetCurrentTsFromPd.
```

Comment:
no need change here

Reply:
oh right, I guess IDE scanned for string match for entire codebase during refactoring